Dockerfile

```diff
@@ -48,10 +48,7 @@ COPY --from=builder --chown=node:node /opt/one-app/development ./
 # production image
 # last so that it's the default image artifact
 FROM node:12-alpine as production
-ENV NODE_ENV=production \
-    HTTPS_PORT=8443 \
```

Comment:
Removing these values as we don't necessarily want to have an opinion on the port numbers our users use and also because by setting them up here prevents the `PORT` env var to override the value from `HTTP_PORT` as required in Heroku and a few other clouds.

Reply:
these pair with the `EXPOSE` statements below https://docs.docker.com/engine/reference/builder/#expose
hm 🤔

Reply:
these pair with the [EXPOSE](https://docs.docker.com/engine/reference/builder/#expose) statements below
>[`EXPOSE`] functions as a type of documentation between the person who builds the image and the person who runs the container, about which ports are intended to be published.

The image should have some defaults IMO but I was also under the impression that each of the env vars could be overridden, including removal. Does Heroku support removing env vars? Does that look like `HTTP_PORT=''`?
hm 🤔
this might be a case where Docker and Heroku disagree about best practice, the downside of following Docker's is the added work/friction for devs to get running on Heroku, the downside of following Heroku's is anything docker-native? but IDK if the hub even uses this information in the UI? ex: https://hub.docker.com/_/nginx

Reply:
No they can't really be removed. This is what heroku's docs have to say about EXPOSE.

https://devcenter.heroku.com/articles/container-registry-and-runtime#dockerfile-commands-and-runtime

EXPOSE is a _suggestion_ per docker docs so I think the right thing to do here is remove EXPOSE as well from the dockerfile.